ocs_ci/ocs/ui/pvc_ui.py

```diff
@@ -65,15 +76,15 @@ def create_pvc_ui(self, sc_type, pvc_name, access_mode, pvc_size, vol_mode):
         logger.info("Create PVC")
         self.do_click(self.pvc_loc["pvc_create"])
 
-        time.sleep(2)
-
     def verify_pvc_ui(self, pvc_size, access_mode, vol_mode, sc_type):
         """
         Verifying PVC details via UI
 
-        pvc_size (str): the size of pvc (GB)
-        access_mode (str): access mode
-        vol_mode (str): volume mode type
+        Args:
+            pvc_size (str): the size of pvc (GB)
+            access_mode (str): access mode
+            vol_mode (str): volume mode type
+            sc_type (str): storage class type
```

Comment:
The comment is not related to this line but could you please add navigation to the specific pvc page so the function is usable from any spot you call it and not just after pvc creation.

Reply:
added, please check

Reply:
Great